certification/internal/shell/has_required_labels.go

```diff
@@ -1,68 +1,45 @@
 package shell
 
 import (
-	"encoding/json"
-	"os/exec"
-
-	"github.com/itchyny/gojq"
 	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification"
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/cli"
 	log "github.com/sirupsen/logrus"
 )
 
 type HasRequiredLabelsCheck struct{}
 
 func (p *HasRequiredLabelsCheck) Validate(image string) (bool, error) {
 	// TODO: if we're going have the image json on disk already, we should use it here instead of podman inspect-ing.
-	stdouterr, err := exec.Command("podman", "inspect", image).CombinedOutput()
+	labels, err := p.getDataForValidate(image)
 	if err != nil {
 		log.Error("unable to execute inspect on the image: ", err)
```

Comment:
We're double logging here, right? `p.getDataForValidate`, if it encounters an error, will log with the same message. Either this needs to be a different message, or we don't need to log it here, or downstream.